src/shippingservice/src/telemetry/resources_conf.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use opentelemetry_sdk::{
+    resource::{
+        EnvResourceDetector, OsResourceDetector, ProcessResourceDetector, ResourceDetector,
+        SdkProvidedResourceDetector, TelemetryResourceDetector,
+    },
+    Resource,
+};
+use std::time::Duration;
+
+pub fn get_resource_attr() -> Resource {
+    let os_resource = OsResourceDetector.detect(Duration::from_secs(0));
+    let process_resource = ProcessResourceDetector.detect(Duration::from_secs(0));
```

Comment:
Not for now, just fyi - this would be a breaking change in future as OsResourceDetector and ProcessResourceDetector are going to move out of otel-sdk to separate crate - https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/49

Reply:
Great to know beforehand!
Thank you!!!